crates/ruff_db/src/panic.rs

```diff
@@ -21,7 +21,7 @@ thread_local! {
 
 /// [`catch_unwind`](std::panic::catch_unwind) wrapper that sets a custom [`set_hook`](std::panic::set_hook)
 /// to extract the backtrace. The original panic-hook gets restored before returning.
-pub(crate) fn catch_unwind<F, R>(f: F) -> Result<R, PanicError>
+pub fn catch_unwind<F, R>(f: F) -> Result<R, PanicError>
```

Comment:
All of the chatter in this crate is just to publicize this existing helper function.